civihr_hrjobcontract_entities/civihr_hrjobcontract_entities.module

```diff
@@ -251,6 +251,43 @@ function civihr_hrjobcontract_entities_init() {
                     WHERE c.deleted = 0
                     ORDER BY t.job_contract_id");
 
+
+        db_query('DROP VIEW IF EXISTS hrjc_contact_details');
+        db_query("CREATE OR REPLACE VIEW hrjc_contact_details AS
+          SELECT c.id AS contact_id,
+            GROUP_CONCAT(DISTINCT CASE WHEN phone_location.name = 'Work' THEN c_phone.phone ELSE '' END
+              ORDER BY c_phone.phone DESC SEPARATOR ' ') AS work_phone,
+            GROUP_CONCAT(DISTINCT CASE WHEN phone_location.name = 'Work' THEN c_phone.phone_ext ELSE '' END 
+              ORDER BY c_phone.phone_ext DESC SEPARATOR ' ') AS work_phone_ext,
+            GROUP_CONCAT(DISTINCT CASE WHEN email_location.name = 'Work' THEN c_email.email ELSE '' END 
+              ORDER BY c_email.email DESC SEPARATOR ' ') AS work_email,
+
+            GROUP_CONCAT(DISTINCT CASE WHEN phone_location.name = 'Home' THEN c_phone.phone ELSE '' END
```

Comment:
I think you can use IN statement above instead of duplicating them for work and home .. though as I said I don't think we need home phone and email at all (to be checked with mike)